src/mmw/mmw/settings/layer_settings.py

```diff
@@ -145,6 +145,30 @@
         # overlaps with perimeter polygon.
     },
     {
+        'code': 'nhd_quality_tp',
+        'display': 'Delaware River Basin TP Concentration',
```

Comment:
@mmcfarland is it a viable option to quietly show two layers (NHD and then water quality) when the water quality overlay is enabled? It avoids the left join and might still be somewhat fast if the NHD tiles are cached.